### ISSUE TITLE
bug: ion-content: Scroll not working for android 4.4.2.

### ISSUE THREAD

**Issue by fasilpv** [driftyco/ionic#5179]
**Type**: <span ionic-type>bug</span>

**Ionic Version**: <span ionic-version>1.x</span>

**Platform**: <span ionic-platform>android</span> <span ionic-platform-version>4.4</span>  <span ionic-webview>webview</span>

<span ionic-description>Hi there,

This is my structure

ion-content -> ion-list -> ion-item -> div -> div.

I am using ng-repeat as well.

I can see the ionic scroll working on Android 5.1.2 version, but the same is not working for Android version 4.4.2. Am I missing something? Is this a known issue?

This is ionic info

Cordova CLI: 5.4.1
Gulp version: CLI version 3.9.0
Gulp local: Local version 3.9.0
Ionic Version: 1.2.4
Ionic CLI Version: 1.7.13
Ionic App Lib Version: 0.6.5
ios-deploy version: 1.8.4 
ios-sim version: 5.0.6 
OS: Mac OS X El Capitan
Node Version: v4.2.4
Xcode version: Xcode 7.1 Build version 7B91b

Thanks</span>

<span is-issue-template></span>

**Comment by HanxiongShi** [comment]
+1

**Comment by CarlosAmaral** [comment]
+1

**Comment by phodal** [comment]
I have a scroll issue, and I try to downgrade ionic to 1.2.3. And I found the bug was fixed.

**Comment by CarlosAmaral** [comment]
overflow-scroll:false in the ion-content worked for me. Try that.

**Comment by fasilpv** [comment]
That did not work for me. But, it worked when I removed the style 'display: table !important;' from .scroll-content.  Thank you.

**Comment by Moes3s** [comment]
+ 1. Works fine on 5.1.1 and iOs 9

**Comment by varunis** [comment]
Setting overflow-scroll="false" worked for me too. Thanks @CarlosAmaral !
<ion-content overflow-scroll="false">

**Comment by CarlosAmaral** [comment]
You're welcome @varunis

**Comment by cnyunguo** [comment]
@varunis thanks 谢谢你解决了我的问题

**Comment by appplumbr** [comment]
@All, overflow-scroll="false" in ion-content worked for me for android 4.4

**Comment by kevinokeyo** [comment]
overflow-scroll="false" in ion-content worked for me too thanks @CarlosAmaral

**Comment by adrianoresende** [comment]
+1 
overflow-scroll="false" worked for me but this has bug in browser too

**Comment by danbucholtz** [comment]
Hi @fasilpv,

Can you provide a codepen or plunkr?  I cannot recreate with this:

```
<ion-content overflow-scroll="true">
      <!--<ion-refresher on-refresh="doRefresh()">

      </ion-refresher>-->
      <ion-list>
        <ion-item ng-repeat="item in items">
          <div>
              <div>{{item}}</div>
          </div>
        </ion-item>
      </ion-list>
    </ion-content>
```

**Comment by jgw96** [comment]
Hello! As it has been a little while since there was any activity on this issue i will be closing for now, but, if you are able to get that codepen/plunkr together feel free to comment!

**Comment by giogit** [comment]
The issue is still present with Android 4.3 (api 18). You can reproduce it also on AVD.
I tried the workaround (overflow-scroll:false on ion-content) but the problem persist.

These are environment data:
Cordova CLI: 6.1.1
Gulp version:  CLI version 3.8.11
Ionic Version: 1.3.0
Ionic CLI Version: 1.7.14
Ionic App Lib Version: 0.7.0
AVD with Android 4.3 (api 18)

Thanks

**Comment by jgw96** [comment]
Hello @giogit ! Would you mind trying Crosswalk with your app? Although we support Android 4.3 we do recommend using Crosswalk if you plan on your app supporting it.

**Comment by giogit** [comment]
I've just updated my original post, because I made a mistake: if I set overflow-scroll="false" on the ion-content element, the scrolling works on Android 4.3.
I will do some regression test on other platforms.
Thank you for prompt reply!

**Comment by jgw96** [comment]
@giogit No problem! If you run into any issues on other platforms feel free to let me know!

**Comment by alacret** [comment]
Happend to me right now in Android 4.4 - api-19 in the emulator, and Galaxy s4 with Android 4.2 and Galaxy S3 mini with Android 4.2: overflow-scroll="false" does not work for me

**Comment by freddiescott** [comment]
If I set the overflow-scroll="false" then I'm unable to tap/click a button on android 4.2, emulators and real devices.

**Comment by dovk** [comment]
I found that placing `overflow-scroll="false"` in <ion-content> and defining <ion-scroll> as 
<ion-scroll scrollX="true" scrollY="true" style="margin:17rem;position:fixed;display:inline;top:0;right:0;bottom:0;left:0; white-space: nowrap;">`
helped - I found that I could not just swipe up or down - if there were any input elements along the path, it would not work. But if I would swipe where there were no input elements, or on the side where the scroll bar would have been, it did work. So I brought back the display of the scroll bar on the side, and if the user uses the scroll bar - it works. Not great, but it'll pass for now.
